sdk_extensions/testbed/src/test/java/io/opentelemetry/sdk/extensions/trace/testbed/concurrentcommonrequesthandler/RequestHandler.java

```diff
@@ -30,13 +29,13 @@
 
   private final Tracer tracer;
 
-  private final SpanContext parentContext;
+  private final io.grpc.Context parentContext;
```

Comment:
I wonder if it might be better to rename the `Context` class in this package to `TestContext` or something, so we don't have the name collision in here.

Reply:
I renamed it to RequestHandlerContext in 6eea962. Is that name OK?